packages/sharec-core/lib/steps/readConfigs.js

```diff
@@ -18,12 +19,34 @@ const readConfigs = async (input) => {
   try {
     const configsPaths = await find(configsPath, '**/*')
     const withoutLocks = configsPaths.filter((config) => !/(\.|-)lock/.test(config))
+
+    const withoutBinaries = []
+
+    for (const file of withoutLocks) {
+      if(isText(file) || getEncoding(await readBuffer(file, 24)) === 'utf8') {
+        withoutBinaries.push(file)
+      } else {
+        input.binaries.push({
+          filename: file.replace(configsPath, '').replace(/^\//, ''),
+          original: file
+        })
+      }
+    }
+
     const readedConfigs = {}
 
-    for (const config of withoutLocks) {
+    for (const config of withoutBinaries) {
       const configKey = config.replace(configsPath, '').replace(/^\//, '')
+      const data = await readFile(config, 'utf8')
 
-      readedConfigs[configKey] = await readFile(config, 'utf8')
+      if (data.startsWith('#!')) {
```

Comment:
Files with `#!` – are textual files. They can be processed with common configs flow.
We need to process "really binary" file in additional flow.

Reply:
You want to merge executable script files? Isn't that risky?

Reply:
If we don't have required strategy and file can't be processed as data structure – sharec just makes copy of file, or if file was changed by user since last installation sharec will skip it

Reply:
`#!` are textual files but I've added a `chmod` so executable scripts can still be executable by copying the mode.